concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -0,0 +1,601 @@
+        Concent   -> Provider:    FileTransferToken
+        Provider  -> Concent:     no response
+        Concent   -> Requestor:   ForceGetTaskResultFailed
+        """
+
+        # STEP 1: Requestor forces get task result via Concent.
+        # Concent accepts request if all conditions were met.
+        serialized_task_to_compute = self._get_serialized_task_to_compute(
+            timestamp   = "2017-12-01 10:00:00",
+            task_id     = 99,
+            deadline    = "2017-12-01 11:00:00"
+        )
+        serialized_force_get_task_result = self._get_serialized_force_get_task_result(
+            task_to_compute = serialized_task_to_compute,
+            timestamp       = "2017-12-01 11:00:01",
+        )
+
+        with freeze_time("2017-12-01 11:00:01"):
+            response_1 = self.client.post(
+                reverse('core:send'),
+                data                           = serialized_force_get_task_result,
+                content_type                   = 'application/octet-stream',
+                HTTP_CONCENT_CLIENT_PUBLIC_KEY = self._get_encoded_requestor_public_key()
+            )
+
+        self.assertEqual(response_1.status_code, 200)
+
+        # STEP 2: Provider receives force get task result and file transfer token via Concent.
+        with freeze_time("2017-12-01 11:00:02"):
+            response_2 = self.client.post(
+                reverse('core:receive'),
+                content_type                   = 'application/octet-stream',
+                HTTP_CONCENT_CLIENT_PUBLIC_KEY = self._get_encoded_provider_public_key()
+            )
+
+            response_3 = self.client.post(
+                reverse('core:receive'),
+                content_type                   = 'application/octet-stream',
+                HTTP_CONCENT_CLIENT_PUBLIC_KEY = self._get_encoded_provider_public_key()
+            )
+
+        self.assertEqual(response_2.status_code, 200)
+        self.assertEqual(response_3.status_code, 200)
+
+        message_1_from_concent = load(response_2.content, self.PROVIDER_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
+        message_2_from_concent = load(response_3.content, self.PROVIDER_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
+
+        # Assign each message to correct variable
+        if isinstance(message_1_from_concent, message.ForceGetTaskResult):
+            message_force_get_task_result = message_1_from_concent
+            message_file_transfer_token = message_2_from_concent
+        else:
+            message_force_get_task_result = message_2_from_concent
+            message_file_transfer_token = message_1_from_concent
+
+        # Test ForceGetTaskResult message
+        self.assertIsInstance(message_force_get_task_result, message.ForceGetTaskResult)
+        self.assertEqual(message_force_get_task_result.timestamp, self._parse_iso_date_to_timestamp("2017-12-01 11:00:02"))
+        self.assertTrue(message_force_get_task_result.task_to_compute)
```

Comment:
`task_to_compute` is not a boolean. You should use `assertEqual()` and compare it with a specific value.